tests/integration/test_aroi_pagination.py

```
@@ -150,13 +159,13 @@ def test_pagination_css_classes(self):
         
         # Test pagination-section class exists for all pages
         pagination_sections = re.findall(r'class="pagination-section"', rendered)
-        # Should have 3 sections × 12 categories = 36 pagination sections
-        self.assertEqual(len(pagination_sections), 36)
+        # Should have 3 sections × 14 categories = 42 pagination sections
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)